specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -278,27 +278,27 @@ finally, commit the transaction.
 
 === Transaction Interoperability
 
-=== Multiple Java EE Platform Interoperability
+=== Multiple Jakarta EE Platform Interoperability
 
 This specification does not require the Product
 Provider to implement any particular protocol for transaction
-interoperability across multiple Java EE products. Java EE compatibility
-requires neither interoperability among identical Java EE products from
-the same Product Provider, nor among heterogeneous Java EE products from
+interoperability across multiple Jakarta EE products. Jakarta EE compatibility
+requires neither interoperability among identical Jakarta EE products from
+the same Product Provider, nor among heterogeneous Jakarta EE products from
 multiple Product Providers.
 
-We recommend that Java EE Product Providers use
+We recommend that Jakarta EE Product Providers use
 the IIOP transaction propagation protocol defined by OMG and described
 in the OTS specification (and implemented by the Java Transaction
 Service), for transaction interoperability when using the EJB
```

Comment:
Probably need to modify EJB to Jakarta Enterprise Beans

Reply:
Resolved